controllers/utils.go

```diff
@@ -91,3 +92,60 @@ func removeCondition(
 	obj.SetConditions(conditions)
 	return r.Update(ctx, obj)
 }
+
+func deleteAllCustomResources(ctx context.Context, c client.Client, namespace string) error {
+	opts := []client.DeleteAllOfOption{
+		client.InNamespace(namespace),
+		client.GracePeriodSeconds(5),
+	}
+	// Delete all users
+	users := apiV1.PgUserList{}
+	if err := c.List(ctx, &users); err != nil {
+		return nil
+	}
+
+	for _, user := range users.Items {
+		user.Finalizers = []string{}
+		if err := c.Update(ctx, &user); err != nil {
+			return err
+		}
+	}
+	user := apiV1.PgUser{}
+	if err := c.DeleteAllOf(ctx, &user, opts...); err != nil {
```

Comment:
This can happen in a real environment.
This function is intended to be only used in tests.
Is there any way to share functions only with the tests in a package?